.jshintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "globals"     : { "Phaser": false, "PIXI": false, "p2": false, "CocoonJS": false, "process": false, "game": true },
+  "globals"     : { "Phaser": false, "PIXI": false, "p2": false, "CocoonJS": false, "process": false, "game": true, "randomTexture": true, "smoothstep": true },
```

Comment:
Check out the last travis build before this one. It threw an error cause it said I was redefining the variables that I had declared in the global scope. So I thought I needed to set them to true.